bin/check-marathon-task.rb

```diff
@@ -173,4 +168,17 @@ def check_tasks(data)
 
     [tasks.length, unhealthy]
   end
+
+  def default_tasks(server)
+    expected_tasks_url = "/v2/apps/#{config[:task]}"
+    url = URI.parse("#{config[:protocol]}://#{server}:#{config[:port]}#{expected_tasks_url}")
+    req = Net::HTTP::Get.new(url)
```

Comment:
Any particular reason why was not the RestClient::Resource used? Would be more consistent as most of the scripts use that.

Reply:
Just reused the method already existing in the plugin, I think maybe using RestClient::Resource for the new functionality wouldn't be consistent, so it might require to rewrite more stuff within the plugin to produce a consistent code.

Reply:
I agree that we should stick to what the plugin is already using. I we want to change it tp use `RestClient::Resource` we should change it everywhere in the plugin.

Reply:
actually, I've now checked some of the other scripts and they are using `RestClient::Resource` so I think we should use that here as well, not a blocker but we should try to keep consistency.